pennylane/ops/qubit.py

```diff
@@ -919,6 +976,9 @@ def decomposition(theta, wires):
         for i in range(len(wires) - 1):
             CNOT(wires=[wires[i + 1], wires[i]])
 
+    def adjoint(self, do_queue=False):
+        raise NotImplementedError
```

Comment:
Is this because it is non-trivial to invert, or that we can't represent the inverse in PL currently? (does `MultiRZ(..).inv()` work?)

Reply:
I left a bunch as NotImplemented just to get the draft out sooner. I'll be sure to implement all of them when this becomes a real PR.